src/ORM/Behavior/TranslateBehavior.php

```diff
@@ -110,10 +111,12 @@ public function initialize(array $config)
             $this->_translationTable = TableRegistry::get($translationAlias);
         }
 
+        $this->config('conditions.model', $this->config('model') ?: $this->config('conditions.model') ?: $this->_table->alias());
```

Comment:
That's why I wanted to remove the `config['model']` .. I don't know why that's necessary since it's settable in the conditions.